src/schema.rs

```diff
@@ -783,3 +783,26 @@ joinable!(version_authors -> users (user_id));
 joinable!(version_authors -> versions (version_id));
 joinable!(version_downloads -> versions (version_id));
 joinable!(versions -> crates (crate_id));
+
+allow_tables_to_appear_in_same_query!(
```

Comment:
Hopefully this is less confusing now

Reply:
If we generate it for all tables, why do we have to explicitly list this at all...?

Reply:
I don't understand the question

Reply:
> we just speculatively generate it for all tables

By "we", do you mean "crates.io" or "diesel"? If Diesel is doing this speculative generation, why does it have to be listed explicitly in src/schema.rs at all? Why isn't this built-in to diesel?

Reply:
This file was generated by `diesel print-schema`